pkgs/leak_tracker_testing/lib/src/leak_testing.dart

```diff
@@ -121,6 +121,10 @@ class LeakTesting {
   /// In the result the ignored limit for a class is the
   /// maximum of two original ignored limits.
   /// Items in [classes] will be added to all ignore lists.
+  ///
+  /// Setting [createdByTestHelpers] to true may cause significant
+  /// performance impact on the test run, caused by conversion of
+  /// creation call stack to String.
```

Comment:
is there any way to improve the performance here instead of just documenting that it is poor?

Reply:
I did not find. We can brainstorm it together :)